boards/arm/nucleo_l4r5zi/nucleo_l4r5zi.yaml

```diff
@@ -17,5 +17,6 @@ supported:
   - usb device
   - nvs
   - counter
+  - lptim
```

Comment:
Please also add this to all other boards that would support this feature.

Reply:
For the moment only this one was actually tested. It would be easy though to include for any board where the soc supports the feature.